src/common/Input.tsx

```diff
@@ -34,36 +37,52 @@ const Input: React.FC<InputProps> = (props) => {
   const hasHelperText = !!(helper || errorMessage);
   const hasMaxLength = !!maxLength;
 
+  const [passwordVisible, setPasswordVisible] = React.useState(false);
+
   return (
     <div
       className={classnames("input", {
-        error: !!errorMessage,
+        "error": !!errorMessage,
```

Comment:
No need to surround these with `"`. The reason the others are is because they contain `-`s. Those are invalid JS names, and thus need to be quoted as strings to be used as keys.

Reply:
The Prettier config woes 🙈